spec/acceptance/multiple_definition_functions_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "acceptance_helper"
+
+describe "Multiple definition functions" do
+  before(:each) do
```

Comment:
I generally shy away from `before`, `around` and `after` blocks. And would probably prefer if were to do so here too and inline the creation of each function in their own test.

We would decrease the duplication if the functions did not re-use themselves between each other.